geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateGatewaySenderCommand.java

```diff
@@ -63,7 +67,7 @@ public Result createGatewaySender(@CliOption(key = {CliStrings.GROUP, CliStrings
 
       @CliOption(key = CliStrings.CREATE_GATEWAYSENDER__PARALLEL, specifiedDefaultValue = "true",
           unspecifiedDefaultValue = "false",
```

Comment:
this boolean would never be null, so it's better to have it remain as primitive.